spec/moleculer/broker_spec.rb

```diff
@@ -32,13 +32,76 @@ def test_event; end
     Moleculer::Broker.new(Moleculer::Configuration.new(
                             node_id:   "test1",
                             services:  [service_1, service_2],
-                            log_level: "trace",
+                            logger: false,
+                            transporter: "fake://localhost"
                           ))
   end
 
+  let(:transporter) { subject.instance_variable_get(:@transporter) }
+
   before :each do
+    allow(transporter).to receive(:publish)
     allow(event_1).to receive(:execute).with(packet.data, subject)
     allow(event_2).to receive(:execute).with(packet.data, subject)
+    subject.start
+  end
+
+  describe "#start" do
+    let(:registry) { subject.instance_variable_get(:@registry) }
+    let(:local_node) { registry.local_node }
+
+    it "starts the broker" do
+      expect(subject.started?).to be_truthy
+    end
+
+    it "registers the local node" do
+      expect(subject.local_node).to eq(local_node)
+    end
+
+    it "subscribes to all of the transporter channels" do
+      expect(transporter.subscriptions["MOL.INFO.#{local_node.id}"]).to_not be_nil
```

Comment:
Would it be worth looping over the different types of transporter channels since they all have the same expectation?
```
types.each do |t|
  expect(transporter.subscriptions[t]).to_not be_nil
end
```
or we could also look at something closer to this. 
```
types.all? { |t| expect(transporter.subscriptions[t]).to_not be_nil }
```

I'm not partial either way, the verbosity is nice but this is also not a hard spec to understand.